libs/core/src/lib/token/tokenizer.component.html

```diff
@@ -1,14 +1,11 @@
 <div
     class="fd-tokenizer fd-tokenizer--scrollable"
-    tabindex="0"
-    (focus)="tokenizerHasFocus = true"
-    (blur)="tokenizerHasFocus = false"
-    [ngClass]="{ 'is-focus': tokenizerHasFocus, 'fd-tokenizer--compact': compact }"
+    [ngClass]="{ 'is-focus': _tokenizerHasFocus, 'fd-tokenizer--compact': compact }"
```

Comment:
I think we decided to follow the format:
[class.is-focus] = "_tokenizerHasFocus"
[class.fd-tokenizer--compact] = "compact"